maintainer/README.md

```diff
@@ -0,0 +1,2 @@
+The folder contains an ad-hoc collection of guides, policies and rules of thumb
```

Comment:
As said on slack, I don't think we want to create this here unless we want to move things out of the google maintainers doc we've been using and I'd appreciate a for/against poll on #maintainers before doing that switch.  I've done well over half the edits to that doc and find PRs significantly (and annoyingly) more time consuming hence my preference, but if other maintainers would find this more discover-able by all means kick it off.

Reply:
fwiw the `maintainer/` dir was added in #25596 - i did try to flag that there

re using PRs/markdown/public - i think there are quite a lot of benefits - without going into them all here - and im pretty adverse to using word processors - regardless tho, i think the key point is that this info should be public - visible to contributors, prospective maintainers etc

my prefernence would be to find consensus over a poll, and im just wondering whether it needs to be either/or